static/app/views/detectors/hooks/useMetricDetectorAnomalies.spec.tsx

```diff
@@ -68,18 +56,16 @@ describe('useMetricDetectorAnomalies', () => {
       },
     ];
 
-    const {result} = renderHook(
-      () =>
-        useMetricDetectorAnomalies({
-          series,
-          historicalSeries,
-          thresholdType: AlertRuleThresholdType.ABOVE,
-          sensitivity: AlertRuleSensitivity.MEDIUM,
-          interval: 900, // 15 minutes
-          projectId: '1',
-          enabled: true,
-        }),
-      {wrapper: TestContext}
+    const {result} = renderHookWithProviders(() =>
+      useMetricDetectorAnomalies({
+        series,
+        historicalSeries,
+        thresholdType: AlertRuleThresholdType.ABOVE,
+        sensitivity: AlertRuleSensitivity.MEDIUM,
+        interval: 900, // 15 minutes
+        projectId: '1',
+        enabled: true,
+      })
```

Comment:
linter needs to get at this...

```suggestion
    const {result} = renderHookWithProviders(useMetricDetectorAnomalies, {initialProps: {
        series,
        historicalSeries,
        thresholdType: AlertRuleThresholdType.ABOVE,
        sensitivity: AlertRuleSensitivity.MEDIUM,
        interval: 900, // 15 minutes
        projectId: '1',
        enabled: true,
      })
```